misc/connection-limits/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# 0.1.0 - unreleased
```

Comment:
Nit pick, we recommend the below. I don't care which syntax we use, as long as we stay consistent. Makes my life creating the `docs(CHANGELOG): Prepare xxx` a bit easier.

```suggestion
# 0.1.0 [unreleased]
```

https://github.com/libp2p/rust-libp2p/blob/master/docs/release.md#development-between-releases

Reply:
`[unreleased]` is a link reference that doesn't point to anything so I stopped using that. Can update the docs if you agree.

Reply:
Never thought about that. Yes, patch would be appreciated.

Reply:
PR here: https://github.com/libp2p/rust-libp2p/pull/3615